tox.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
 skipsdist = True
-envlist = py26, py27, py34, pep8
+envlist = py34, py37, py38, pep8
```

Comment:
Do we want to support py35 and/or py36? It has been a few years since I've worked on Python professionally so please excuse me if I'm a little rusty on my understanding of the Python ecosystem.

Reply:
Also, I'm seeing here that Python 3.4 has reached end-of-life. Should we remove it from tox as well? https://www.python.org/downloads/release/python-340/